lib/worker.py

```diff
@@ -219,8 +220,9 @@ def current_task(self, task_id):
                                  task_result, task_tmp_result)
 
     def wrap_result(self, task_id, short_status, result_checksum):
+        is_long = self.suite.check_is_long(task_id[0].split('/', 1)[1])
```

Comment:
Or `os.path.basename(task_id[0])`, it looks a bit simpler.

Reply:
And I would call it from inside the function, just like in `is_test_enabled()`.

Reply:
Right, changed.